README.md

```diff
@@ -64,7 +64,6 @@
 - [ ] **[The user can create a new course by himself]**
 - [ ] **[The app is responsive in Mobile, Tablet and Desktop versions]**
```

Comment:
- [ ]  Please, kindly refactor the main `App` component in `App.js` to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪